st2common/tests/fixtures/print_to_stdout_stderr_sleep.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Copyright 2021 The StackStorm Authors.
+# Copyright 2019 Extreme Networks, Inc.
```

Comment:
That's right. I can confirm that for all the new files we just use the StackStorm Authors and don't need Extreme.


```suggestion
# Copyright 2021 The StackStorm Authors.
```

Reply:
thanks for confirming :+1: